resources/scripts/generate-build-data.sh

```diff
@@ -432,6 +458,7 @@ fetchAndPrepareArtifactsInfo "${ARTIFACTS_INFO}" "${BO_BUILD_URL}/artifacts/" "a
 fetchAndPrepareTestsInfo "${TESTS_INFO}" "${BO_BUILD_URL}/tests/?limit=10000000" "test" "${DEFAULT_LIST}"
 ### fetchAndPrepareTestSummaryReport should run after fetchAndPrepareTestsInfo
 fetchAndPrepareTestSummaryReport "${TESTS_SUMMARY}" "${BO_BUILD_URL}/blueTestSummary/" "test_summary" "${DEFAULT_LIST}" "${TESTS_INFO}"
+fetchAndPrepareTestCoberturaReport "${TESTS_COBERTURA}" "${BUILD_URL}/cobertura/api/json?tree=results\[elements\[name,ratio,denominator,numerator\]\]&depth=3&pretty=true" "test_cobertura" "${DEFAULT_HASH}"
```

Comment:
```suggestion
fetchAndPrepareTestCoberturaReport "${TESTS_COBERTURA}" "${BUILD_URL}/cobertura/api/json?tree=results\[elements\[name,ratio,denominator,numerator\]\]&depth=3" "test_cobertura" "${DEFAULT_HASH}"N
```

Nit: Probably no need to pretty-print this.